certified-connectors/DocuSignDemo/script.csx

```diff
@@ -78,17 +78,18 @@ public class Script : ScriptBase
         { "emailTabs", "Email" },
         { "firstNameTabs", "First Name" },
         { "formulaTabs", "Formula" },
-        { "fullNameTabs", "Name" },
+        { "fullNameTabs", "Full Name" },
         { "initialHereTabs", "Initial" },
         { "lastNameTabs", "Last Name" },
-        { "listTabs", "List" },
+        { "listTabs", "Dropdown" },
```

Comment:
should this be called dropdownTabs ?

Reply:
It is called listTabs but product needs it to be Dropdown for users. Here is the link to listTabs: https://developers.docusign.com/docs/esign-rest-api/reference/envelopes/enveloperecipienttabs/create/#schema__enveloperecipienttabs_listtabs.